editor/test/state.js

```diff
@@ -1033,15 +1032,15 @@ describe( 'state', () => {
 		it( 'should return a redux store', () => {
 			const store = createReduxStore();
 
-			expect( store.dispatch ).to.be.a( 'function' );
-			expect( store.getState ).to.be.a( 'function' );
+			expect( typeof store.dispatch ).toBe( 'function' );
+			expect( typeof store.getState ).toBe( 'function' );
 		} );
 
 		it( 'should have expected reducer keys', () => {
 			const store = createReduxStore();
 			const state = store.getState();
 
-			expect( Object.keys( state ) ).to.have.members( [
+			expect( Object.keys( state ) ).toEqual( expect.arrayContaining( [
```

Comment:
The `toEqual( expect.someWeirdObject` syntax in general is strange.  Again not something we can really change here.